Libraries/LibGUI/AbstractView.cpp

```diff
@@ -262,6 +262,10 @@ void AbstractView::mousemove_event(MouseEvent& event)
     if (!model())
         return ScrollableWidget::mousemove_event(event);
 
+    auto data_type = m_model->drag_data_type();
+    if (data_type.is_null())
+        return ScrollableWidget::mousemove_event(event);
+
     auto hovered_index = index_at_event_position(event.position());
     set_hovered_index(hovered_index);
```

Comment:
We still need to update the hovered index, even if we don't want to initiate a drag.
So the early return needs to occur after the call to `set_hovered_index()`

Also the commit message is very sloppy, please proofread and tidy it up :)